docs/src/_guide/orchestration.md

```diff
@@ -27,6 +29,18 @@ meltano schedule add carbon__sqlite --extractor tap-carbon-intensity --loader ta
 
 Now that you've scheduled your first pipeline, you can load the "Pipeline" page in the UI and see it show up.
 
+### Scheduling predefined jobs
```

Comment:
Can we reorder these sections, since `meltano run`/jobs are now preferred over `meltano elt`?